content/awell-orchestration/api-reference/queries/get-form.mdx

```diff
@@ -20,11 +20,14 @@ Forms in Awell can have questions with display / visibility logic. Meaning that
 query GetForm($id: String!) {
   form(id: $id) {
     form {
-      id
+      id # static id from Studio
+      definition_id
```

Comment:
Isn't it the inverse?
- `id` = published id
- `definition id` = static id form Awell Studio

Reply:
you're right, good catch!